data_import/models.py

```diff
@@ -198,12 +198,22 @@ class NewDataFileAccessLog(models.Model):
     data_file = models.ForeignKey(
         DataFile, related_name="access_logs", on_delete=models.SET_NULL, null=True
     )
+    serialized_data_file = JSONField(default=dict, null=True)
     data_file_key = JSONField(default=dict, null=True)
     aws_url = models.CharField(max_length=400, null=True)
 
+    @property
+    def get_data_file(self):
+        """
+        Helper that returns a queryset with the DataFile if it exists still, empty if not.
+        """
+        return DataFile.objects.filter(
+            id=self.serialized_data_file.get("data_file_id", None)
```

Comment:
In the example serialization in the PR description, this appears to be `datafile_id`?

Reply:
In general I think you should consider changing all versions that are "data_file" to "datafile" to be consistent with things like how the serialization treats camelcased models, and the ORM uses all-lower-no-underscore as well as default for references to models.

Reply:
roger that.